blob/fileblob/fileblob.go

```diff
@@ -143,6 +143,7 @@ func (o *URLOpener) forParams(ctx context.Context, q url.Values) (*Options, erro
 	opts := new(Options)
 	*opts = o.Options
 
+	opts.SkipMetadata = (q.Get("skip_metadata") != "")
```

Comment:
Let's improve the documentation for this whole "sidecar" thing. Can you add something above near line 17, something like:

```
//
// fileblob stores blob metadata in sidecar files with an ".attrs" suffix, including
// blob.Attributes fields other than the ones like CreateTime and ModTime
// that are provided by the filesystem.
// This implies that many blob.Attributes fields will set to default values for
// blobs read via fileblob that were not originally written using fileblob.
// Writing of these sidecar files can be suppressed via Options.SkipMetadata.
//
// URLs
```